contracts/meta-oracles/HistoricalPriceFeedV2.sol

```diff
@@ -38,10 +38,19 @@ contract HistoricalPriceFeedV2 is
 {
     using SafeMath for uint256;
 
+    /* ============ Events ============ */
+
+    event LogMedianizerUpdated(
+        address newMedianizerAddress
+    );
+
     /* ============ State Variables ============ */
-    uint256 public updateFrequency;
+    // How often the price feed can be updated in seconds
+    uint256 public updatePeriod;
+    // How far after nextAvailableUpdate timestamp has passed before interpolation is used (in seconds)
     uint256 public updateTolerance;
     uint256 public maxDataPoints;
+    // Timestamp in seconds of next available update time
```

Comment:
While for time intervals above “in seconds” does make sense, for time moments such as this one it is confusing, because timestamp format depends not only on time unit but also on “zero“ time moment.  Usually, people say “timestamp in seconds since epoch” or simply “UNIX timestamp” to describe this format of time moment representation.  Anyway, such format is de-facto standard for Ethereum, so just “timestamp” should be enough.